### PR TITLE
Fix #1350: Swift5 migration

### LINKED ISSUE #1350: Update to Swift5/Xcode11/iOS13
In order to support iOS 13 SDK the app has to be updated to Swift 5 and work on XCode 11

### PR DESCRIPTION
**Pull request by iccub**
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
- [x] My patch or PR title has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!` (or `No Bug: <message>` if no relevant ticket)
- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New files have MPL-2.0 license header.

## Reviewer Checklist:

- [ ] PR is linked to an issue via [Zenhub](https://www.zenhub.com/extension).
- [ ] Issues are assigned to at least one epic.
- [ ] Issues include necessary QA labels:
  - [ ] `QA/(Yes|No)`
  - [ ] `release-notes/(include|exclude)`
  - [ ] `bug` / `enhancement`
- [ ] Necessary security reviews have taken place.
- [ ] Adequate test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable)

### CODE REVIEW

File: BraveShared/Analytics/UrpService.swift [hunk 1]
```diff
@@ -28,17 +28,13 @@ struct UrpService {
...
 
         // Certificate pinning
-        let serverTrustPolicies: [String: ServerTrustPolicy] = [
-            normalizedHost: .pinCertificates(
-                certificates: ServerTrustPolicy.certificates(),
-                validateCertificateChain: true,
-                validateHost: true
-            )
+        let serverTrustPolicies: [String: PinnedCertificatesTrustEvaluator] = [
+            normalizedHost: PinnedCertificatesTrustEvaluator()
```
Comment (Brandon-T) [inline BraveShared/Analytics/UrpService.swift:32]: Should pin public keys of the certificates instead of certificates themselves?

File: Cartfile [hunk 2]
```diff
@@ -1,15 +1,15 @@
-github "Alamofire/Alamofire"		"4.8.2"
+github "Alamofire/Alamofire"		"5.0.0-beta.4"
```
Comment (Brandon-T) [inline Cartfile]: 5.0.0-beta.6 which changes arguments to the `Session`

File: BraveShared/Analytics/UrpService.swift (hunk 1 above)
Comment (iccub) [reply BraveShared/Analytics/UrpService.swift:32]: tbh I don't remember what was the reason for pinning only public keys, the work for it was done over year ago

**Comment by Brandon-T** [comment]
@iccub This PR is supporting Xcode11 but the carthage libraries need Swift-5.0 support?
To fix it, I changed to `SWIFT_VERSION=5.0` in the `./bootstrap.sh` file.

**Comment by Brandon-T** [comment]
@iccub Change to `SWIFT_VERSION=5.0` in the `./bootstrap.sh` file and `KIF` framework to `v3.7.8` in the Carthage file.

**Comment by iccub** [comment]
Done @Brandon-T :)

**Comment by jhreis** [comment]
Lots of deprecation warnings:

Screen Shot 2019-08-19 at 7 57 20 AM

**Comment by jhreis** [comment]
I am still going to merge this, and created follow up ticket: https://github.com/brave/brave-ios/issues/1409

**Review by jhreis — APPROVED** [review]

**Pull request merged**